test/memory/memory_api_test.go

```diff
@@ -155,7 +155,8 @@ func BenchmarkAPIMemory(b *testing.B) {
 		}
 	}
 	if !pass {
-		b.Fatal("one or more memory thresholds was exceeded")
+		b.Errorf("one or more memory thresholds was exceeded")
+		b.FailNow()
```

Comment:
What is the difference?  

[`func (*B) Fatalf`](https://pkg.go.dev/testing#B.Fatalf)
> Fatalf is equivalent to Logf followed by FailNow.